src/PointwiseFunctions/Hydro/EquationsOfState/Enthalpy.cpp

```diff
@@ -0,0 +1,451 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/Hydro/EquationsOfState/Enthalpy.hpp"
+
+#include <cmath>
+#include <functional>
+#include <numeric>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "NumericalAlgorithms/RootFinding/NewtonRaphson.hpp"
+#include "Parallel/Printf.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+namespace EquationsOfState {
+template <typename LowDensityEoS>
+Enthalpy<LowDensityEoS>::Coefficients::Coefficients(
+    std::vector<double> in_polynomial_coefficients,
+    std::vector<double> in_sin_coefficients,
+    std::vector<double> in_cos_coefficients, double in_trig_scale,
+    double in_reference_density, double in_exponential_constant)
+    : polynomial_coefficients(std::move(in_polynomial_coefficients)),
+      sin_coefficients(std::move(in_sin_coefficients)),
+      cos_coefficients(std::move(in_cos_coefficients)),
+      trig_scale(in_trig_scale),
+      reference_density(in_reference_density) {
+  if (not std::isnan(in_exponential_constant)) {
+    // In practice we should always know this at compile time,
+    // it should probably be templated on
+    has_exponential_prefactor = true;
+    exponential_external_constant = in_exponential_constant;
+  } else {
+    has_exponential_prefactor = false;
+    // Maybe this should be a signaling nan?
+    exponential_external_constant = std::numeric_limits<double>::quiet_NaN();
+  }
+}
+
+// Given an expansion h(x) = sum_i f_i(x) , compute int_a^x sum_i f_i(x) e^x +
+// F(a)
+template <typename LowDensityEoS>
+typename Enthalpy<LowDensityEoS>::Coefficients
+Enthalpy<LowDensityEoS>::Coefficients::compute_exponential_integral(
+    const std::pair<double, double>& initial_condition) {
+  // Not yet implemented
```

Comment:
Yeah sorry this was a leftover comment when I thought that this was something that should be implemented (I can't conceive of a reason it should be at this point).  I'll add the above comment just in case